deepchem/utils/poly_wd_graph_utils.py

```diff
@@ -0,0 +1,153 @@
+from typing import List, Tuple
+import re
+
+
+class PolyWDGStringValidator():
+    """
+    Class for validating the string format of weighted directed graph data.
+    This class provides methods to validate the format of a datapoint string.
+    This is a specific string format that is used for storing weighted directed polymer data in a parsable format.
```

Comment:
pls move next line after 70 characters.

Reply:
At few places (like code and the example format) its ok to go beyond 70 characters but mostly we prefer less than 70 per line in normal text.

Reply:
done